packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -169,9 +169,10 @@ export class CommandBarBase extends React.Component<ICommandBarProps, {}> implem
       onClick: this._onButtonClick(item),
     };
 
-    if (item.iconOnly && itemText !== undefined) {
+    if (item.iconOnly && (itemText !== undefined || (item.tooltipHostProps && item.tooltipHostProps.content))) {
```

Comment:
I feel like we could simplify this just to check for tooltipHostProps instead of tooltipHostProps and tooltipHostProps.content. Thinking about scenarios where someone passes in only onRenderContent.

Reply:
True, but it could be possible to not send anything too and may end up showing a blank content. Instead we should check on onRenderContent as well. What do you think?

Reply:
It would be weird for someone to pass tooltipHostProps and not want to display it no? I just don't think we need to box it in like that. It doesn't seem an unsafe assumption to make that if someone passes this optional prop in that they want a tooltip.